BrandonBeltran/hackerrank/03/JavaMap.java

```diff
@@ -0,0 +1,34 @@
+package bootcamp;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class JavaMap {
+
+    public static void main(String[] args) {
+
+        Map<String, Integer> phoneBook = new HashMap<>();
```

Comment:
Best practice in Collections or Map Variable Name, is to use singular name and interface name: numberList, change it in all variables